common/reviews/api/heft-config-file.api.md

```diff
@@ -131,34 +148,38 @@ export class NonProjectConfigurationFile<TConfigurationFile> extends Configurati
     loadConfigurationFileAsync(terminal: ITerminal, filePath: string): Promise<TConfigurationFile>;
     tryLoadConfigurationFile(terminal: ITerminal, filePath: string): TConfigurationFile | undefined;
     tryLoadConfigurationFileAsync(terminal: ITerminal, filePath: string): Promise<TConfigurationFile | undefined>;
-    // (undocumented)
-    protected _tryLoadConfigurationFileInRig(terminal: ITerminal, rigConfig: IRigConfig, visitedConfigurationFilePaths: Set<string>): TConfigurationFile | undefined;
-    // (undocumented)
-    protected _tryLoadConfigurationFileInRigAsync(terminal: ITerminal, rigConfig: IRigConfig, visitedConfigurationFilePaths: Set<string>): Promise<TConfigurationFile | undefined>;
 }
 
+// @beta
+export const PathResolutionMethod: {
+    readonly resolvePathRelativeToConfigurationFile: "resolvePathRelativeToConfigurationFile";
+    readonly resolvePathRelativeToProjectRoot: "resolvePathRelativeToProjectRoot";
+    readonly NodeResolve: "NodeResolve";
+    readonly nodeResolve: "nodeResolve";
```

Comment:
Is this for legacy reasons?

Reply:
I didn't want to make a breaking change in this API during this change, so I kept the signature, just allowed use of a string literal instead of calling the enum.